api/features/tasks.py

```diff
@@ -38,19 +36,16 @@ def trigger_feature_state_change_webhooks(
     previous_state = _get_previous_state(history_instance, event_type)
     if previous_state:
         data.update(previous_state=previous_state)
-    Thread(
-        target=call_environment_webhooks,
-        args=(instance.environment, data, event_type),
-    ).start()
 
-    Thread(
-        target=call_organisation_webhooks,
+    call_environment_webhooks.delay(args=(instance.environment, data, event_type))
```

Comment:
As task arguments are serialised in the database using JSON, these arguments need to be JSON-serialisable. We will definitely need to change `instance.environment` to `instance.environment_id` and (I think) we will also need to change `event_type` to `event_type.value`. Obviously this will also need changes to the `call_environment_webhooks` (and `call_organisation_webhooks` functions). 

```python
call_environment_webhooks.delay(args=(instance.environment_id, data, event_type.value))
```

Reply:
Made the arguments serializable.